app/view_models/prior_authority/service_costs_with_adjustments.rb

```diff
@@ -0,0 +1,139 @@
+module PriorAuthority
+  module RequestedServiceCosts
+    def requested_humanized_units
+      if cost_type == 'per_item'
+        "#{requested_items} " \
+          "#{I18n.t("prior_authority.application_details.items.#{item_type}").pluralize(requested_items)}"
+      else
+        format_period(requested_period)
+      end
+    end
+
+    def requested_humanized_cost_per_unit
+      i18n_key = cost_type == 'per_item' ? "per_#{item_type}" : cost_type
+
+      "#{requested_base_cost_per_unit} " \
+        "#{I18n.t(i18n_key, scope: 'prior_authority.application_details.items.per_unit_descriptions')}"
+    end
+
+    def requested_formatted_service_cost_total
+      NumberTo.pounds(requested_base_cost)
+    end
+
+    private
+
+    def requested_items
```

Comment:
There are a few of these. Might be neater to use `alias_attribute`/`alias_method`. But equally, do we really need to create all these additional names for existing methods?